isisd/isis_snmp.c

```diff
@@ -35,6 +35,7 @@
 #include "command.h"
 #include "memory.h"
 #include "smux.h"
+#include "zclient.h"
```

Comment:
Hm, looking again, isn't this included in `#include "lib/zclient.h` (below)?

Reply:
exactly. someone else did the job for me.
I close the ticket.